src/server/validation/validateRequest.test.ts

```diff
@@ -52,3 +55,103 @@ test('validateRequest', async () => {
     [1] "pw" length must be at least 8 characters long"
   `)
 })
+
+describe('validateRequest.headers', () => {
+  let app: ExpressApp
+  interface TestResponse {
+    ok: 1
+    headers: StringMap<any>
+  }
+
+  beforeAll(async () => {
+    const resource = getDefaultRouter()
+    resource.get('/', async (req, res) => {
+      validateRequest.headers(
+        req,
+        objectSchema<any>({
+          shortstring: stringSchema.min(8).max(16),
+          numeric: numberSchema,
+          bool: stringSchema,
```

Comment:
The `booleanSchema` does no casting, it simply screams in horror when it sees a string.